xDripG5/Messages/CalibrationDataRxMessage.swift

```diff
@@ -18,5 +21,8 @@ struct CalibrationDataRxMessage: TransmitterRxMessage {
         guard data.starts(with: .calibrationDataRx) else {
             return nil
         }
+
+        glucose = data[11..<13].toInt()
```

Comment:
Are you sure the first bit isn't used for something else like in `GlucoseRxMessage`?

Reply:
You mean the bytes 1 to 10? Yes I am sure they are used for something, I just can't work out what that is. Here's an example: `3300352d0090012d00da00 cb00 3bce5100 9e19`. There's nothing in the bytes 1 to 10 that looks like timestamp, status, or anything else that's familiar. I was hoping to be able to extract calibration parameters from these bytes (e.g. slope and offset), but they change every read despite having the same last calibration.

Reply:
OK, good question. It's the 4 bits with highest order, isn't it, going by the mask `glucoseIsDisplayOnly = (glucoseBytes & 0xf000) > 0`?

They could certainly be used for something else, but I don't know what that would be. This is a meter BG, so "display-only" doesn't make sense here. To be safe, we could use a mask (`glucoseBytes & 0xfff`). These transmitter message aren't particularly optimized for size, so it would surprise me if they tried to fit something else in those 4 bits.

I also can't think of a reliable way to test. All I can say is I have never seen those 4 bits set to anything other than zero.

Reply:
Yep – we should go ahead and mask the value. Glucose values will never use those bits anyway.